build_runner/bin/graph_inspector.dart

```diff
@@ -100,6 +100,7 @@ class InspectNodeCommand extends Command {
         ..writeln('  type: ${node.runtimeType}');
 
       if (node is GeneratedAssetNode) {
+        description.writeln('  needsUpdate: ${node.needsUpdate}');
```

Comment:
Is this just logging inf?

Reply:
ya, this is just outputting an additional field when you query for a node